src/main/java/seedu/guest/logic/commands/MarkRoomsUncleanCommand.java

```diff
@@ -18,17 +18,17 @@
 
 
 /**
- * Changes isRoomClean field of all guest in current list to "no".
+ * Changes isRoomClean field of all guest in guest book to "no".
```

Comment:
Should this be `guests` instead of `guest`?

Reply:
Fixed!